### ISSUE TITLE
zenga.hu: breakage

### ISSUE THREAD

**Issue by zolikonta** [uBlockOrigin/uAssets#25694]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I am not using uBO Lite along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBOL Lite and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://www.zenga.hu/`

### Category

breakage

### Description

Site does not go beyond a loading screen. Disabling uBOL addon or setting it to no filtering mode fixes it.

### Other extensions used

 none

### Configuration

```yaml
version: uBOL 2024.10.14.189
mode: basic / basic
rulesets:
 default
 hun-0
```

**Comment by stephenhawk8054** [comment]
You'll need to set `optimal` or `complete` mode by default to unbreak the site.

**Comment by zolikonta** [comment]
> You'll need to set `optimal` or `complete` mode by default to unbreak the site.

Thank you for the quick response.

Although that fixed the issue I would like to mention that it was not clear for me that a "higher" filtering setting would help a completly blocked(?) filtered(?) site. I thought I need "less filters" to get it work again. I imagined that the higher the setting the more content it filters. Turns out it's not the case, a higher setting would make more sophisticated filtering. Maybe its the scale UI that made me assume things. I would make it more clear somehow or maybe it's just me not understanding.

Either way keep up the good work and thank you again.
